etc/ReplayOfflineConfiguration.py

```diff
@@ -1640,6 +1655,11 @@
 ignoreStream(tier0Config, "streamL1Rates")
 ignoreStream(tier0Config, "streamDQMRates")
 
+ignoreStream(tier0Config, "ALCALumiPixelsCountsExpress")
+ignoreStream(tier0Config, "ALCALumiPixelsCountsPrompt")
+ignoreStream(tier0Config, "Calibration")
+ignoreStream(tier0Config, "Express")
+ignoreStream(tier0Config, "NanoDST")
```

Comment:
This was specific to the replay I did, we should not forget to remove these lines at the next full replay